integration/deploy.rb

```diff
@@ -66,13 +66,23 @@ def get_gae_versions
 def build_docker_image app_dir, project_id
   image_name = "google-cloud-ruby-test-%.08x" % rand(0x100000000)
   image_location = "us.gcr.io/#{project_id}/#{image_name}"
+  temp_dockerfile = false
   begin
     # Create default Dockerfile if one doesn't already exist
     if File.file? "Dockerfile"
       fail "The Dockerfile file already exists. Please omit it and try again."
     else
-      FileUtils.cp "#{app_dir}/Dockerfile.example", "Dockerfile"
-      temp_dockerfile = true
+      # Copy example Dockerfile and update with correct content
+      File.open "#{app_dir}/Dockerfile.example" do |source_file|
+        File.open "Dockerfile", "w" do |dest_file|
+          temp_dockerfile = true
```

Comment:
I moved this line inside the block is so the temp file can be deleted even if there's a failure in writing the file. Alternatively, we'll have to make sure the file writing doesn't fail.